presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/ShardCleaner.java

```diff
@@ -330,10 +362,38 @@ void deleteOldCompletedTransactions()
     }
 
     @VisibleForTesting
+    synchronized Set<UUID> getLocalShards()
+    {
+        return storageService.getStorageShards();
+    }
+
+    @VisibleForTesting
+    synchronized void cleanLocalShardsImmediately(Set<UUID> local)
+    {
+        // get shards assigned to the local node
+        Set<UUID> assigned = dao.getNodeShards(currentNode, null).stream()
+                .map(ShardMetadata::getShardUuid)
+                .collect(toSet());
+
+        // mark all local files that are not assigned to deletions
+        Set<UUID> deletions = Sets.difference(local, assigned);
+        if (deletions.isEmpty()) {
```

Comment:
Let's take this out so that so that we always print a log message. This is especially useful when run manually from JMX, so that the admin knows zero things were cleaned vs something went wrong.